treeherder/etl/management/commands/pulse_listener_pushes.py

```diff
@@ -1,37 +1,51 @@
 import environ
 from django.core.management.base import BaseCommand
-
 from treeherder.services.pulse import (PushConsumer,
+                                       TaskConsumer,
                                        prepare_consumers)
 
 env = environ.Env()
 
 
 class Command(BaseCommand):
     """
-    Management command to read pushes from a set of pulse exchanges
+    Management command to read pushes as well jobs from a set of pulse exchanges
 
-    This adds the pushes to a celery queue called ``store_pulse_pushes`` which
-    does the actual storing of the pushes in the database.
+    This adds the pushes to a celery queue called ``store_pulse_pushes`` and ``store_pulse_tasks`` which
+    do the actual storing of the pushes and jobs in the database respectively.
     """
-    help = "Read pushes from a set of pulse exchanges and queue for ingestion"
+    help = "Read pushes and jobs from a set of pulse exchanges and queue for ingestion"
 
     def handle(self, *args, **options):
-        # Specifies the Pulse services from which Treeherder will ingest push
+        # Specifies the Pulse services from which Treeherder will ingest push and will consume task
         # information.  Sources can include properties `hgmo`, `github`, or both, to
         # listen to events from those sources.  The value is a JSON array of the form
         # [{pulse_url: .., hgmo: true, root_url: ..}, ..]
         push_sources = env.json(
             "PULSE_PUSH_SOURCES",
             default=[{"root_url": "https://firefox-ci-tc.services.mozilla.com", "github": True, "hgmo": True, "pulse_url": env("PULSE_URL")}])
 
-        consumers = prepare_consumers(
+        push_consumers = prepare_consumers(
             PushConsumer,
```

Comment:
We need a new consumer than can handle *both* kind of messages.

Reply:
Understood. My consumer is calling one process at a time. Need to even combine my consumer